example_files/visualization/input_visualization_feature.html

```diff
@@ -444,31 +446,80 @@ <h2>Annual End Uses</h2>
           var localEndUseMax = 0;
 
           if(feature['complete_simulation'] == true){
+
             // monthly fuel use
             $scope.monthlyFuelChartData[feature.name] = [];
+            var changeToKbtu = false;
+            var kbtu_datasets = ['Gas:Facility', 'Propane:Facility', 'FuelOil#2:Facility', 'OtherFuels:Facility'];
+            // first iterate through all datasets to see if you'll need to change to kBtu units
             _.forEach(datasets, function (dataset) {
+              if (kbtu_datasets.indexOf(dataset) != -1) {
+                changeToKbtu = true;
```

Comment:
@kflemin I wanted to confirm - the `datasets` variable does not change dynamically based on the scenarioData.js file. Also the scenarioData.js would be reporting the residential fuel types even if there is no consumption but would put 0 for those monthly values.
Would this if statement return `true` always in this case? since these kbtu_datasets are present in the `dataset` array?